replay_processing/sc2files.py

```diff
@@ -95,10 +94,11 @@ def extract_player_data(result, player, logger, debug=False):
     return data
 
 
-def dump(data):
+def dump(data, outfile):
     # TODO: Fix this up
     try:
-        print(",".join([str(data[field]) for field in FIELD_NAMES]))
+        outfile.write(",".join([str(data[field]) for field in FIELD_NAMES]))
+        outfile.write("\n")
     except KeyError:
         from pdb import set_trace
```

Comment:
Can we remove this pdb. We should be using traceback to get failure information, not jumping into a debugger for a KeyError